src/InboundServer/api.yaml

```diff
@@ -686,6 +686,13 @@ paths:
                 middleName: Pierre
                 lastName: Trudeau
               dateOfBirth: '1971-12-25'
+            accounts:
```

Comment:
This is pisp specific. Don't think this should be going into  `master` as of yet

Reply:
but this property is optional

Reply:
Sure that's fine. Don't let it hills back these changes. +1